src/components/common/BusinessContactInfo.vue

```diff
@@ -2,7 +2,7 @@
   <div id="business-contact-info">
      <v-layout row v-if="!isEditing">
         <v-flex md4>
-          <label><strong>Registered Office <br>Contact Information</strong></label>
+          <label><strong>Business Contact<br>Information</strong></label>
```

Comment:
So these changes apply to the other filings as well?

Reply:
From my understanding yes. I will triple check however.

Reply:
Updated!

Reply:
I didn't see a code change for this. So the answer is "yes" ?

Reply:
Um, if the logic the correct way around?

Reply:
Yes indeed. 
Incorporation = `Registered Office...`
Registration = `Business...`
Confirmed with Tracey